src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/index.jsx

```diff
@@ -31,6 +32,9 @@ export default function ConfirmedAppointmentDetailsPage() {
     shallowEqual,
   );
   const appointmentDate = moment.parseZone(appointment?.start);
+  const isVAOSServiceVAAppointments = useSelector(state =>
+    selectFeatureVAOSServiceVAAppointments(state),
+  );
```

Comment:
Step 2. remove these 3 lines and assign the constant to the `getConfirmedAppointmentDetailsInfo` selector function on https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/appointment-list/redux/selectors.js#L254
Step3. Add `usev2` toggle state as its return value. See the added lines in asterisk below:

```
export function getConfirmedAppointmentDetailsInfo(state, id) {
  const { appointmentDetailsStatus, facilityData } = state.appointments;
  **const featureVAOSServiceVAAppointments = selectFeatureVAOSServiceVAAppointments(
    state,
  );**
  return {
    appointment: selectAppointmentById(state, id, [
      APPOINTMENT_TYPES.vaAppointment,
    ]),
    appointmentDetailsStatus,
    cancelInfo: getCancelInfo(state),
    facilityData,
    showCancelButton: selectFeatureCancel(state),
    **useV2: featureVAOSServiceVAAppointments,**
  };
}

```

```

Reply:
@ptouchton Again, I can't highlight on non committed code so I referenced the function and file by the url.